README.md

```diff
@@ -78,6 +80,40 @@ class StaticController < ApplicationController
 end
 ```
 
+`add_tracer` can add a custom span to the specified instance or class method that is already defined. It requires the custom span name and optionally takes the span kind and additional attributes in hash format:
```

Comment:
```suggestion
`add_tracer` can add a custom span to the specified instance or class method that is already defined. It can optional set the span kind and additional attributes provided in hash format:
```

Reply:
The RDoc string made me realize the span name is not required.